README.md

```diff
@@ -126,20 +126,20 @@ Most of the unit tests in Logstash are written using [rspec](http://rspec.info/)
 
 ### Core tests
 
-1- In order to run the core tests, a small set of plugins must first be installed:
+1- To run the core tests you can use the rake task:
 
-    rake test:install-core
-
-2- To run the core tests you can use the rake task:
-
-    rake test:core
+    ./gradlew test
 
   or use the `rspec` tool to run all tests or run a specific test:
```

Comment:
Are there plans to be able to run a single rspec test via gradlew ?

Reply:
@jakelandis hmm not yet, but it seems like that would be pretty cool to have. Then we could drop supporting the `rspec` CLI tool eventually I guess?